Makefile.am

```diff
@@ -259,16 +258,9 @@ CPP_LINT_FILES = $(shell find . \( -name "*.h" -or -name "*.cpp" \) -and ! \( \
         -wholename "./tools/ola_trigger/config.tab.*" -or \
         -wholename "./tools/ola_trigger/lex.yy.cpp" \) | xargs)
 .PHONY : cpplint
-if FOUND_CPPLINT
 cpplint: Makefile.am
-	cpplint.py --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
+if FOUND_CPPLINT
+	cpplint --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
 else
-cpplint: Makefile.am cpplint.py
-	./cpplint.py --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
-endif
-
-if !FOUND_CPPLINT
-cpplint.py:
-	wget -O cpplint.py $(CPP_LINT_URL)
-	chmod u+x cpplint.py
+  $(error cpplint not found. Install cpplint and re-run configure.)
```

Comment:
Can we somehow clarify that it's no longer the Google one? Or does that not matter?

Reply:
Fixed, added note to install from pip.